R/glue-conditions.R

```diff
@@ -0,0 +1,53 @@
+stop_glue <- function(..., .sep = "", .envir = parent.frame(),
```

Comment:
We can talk about this in depth at some point, but for now, I don't think this approach is something we should be copying going forward.